atom/renderer/api/atom_api_spell_check_client.cc

```diff
@@ -171,7 +178,13 @@ bool SpellCheckClient::IsValidContraction(const SpellCheckScope& scope,
   int word_start;
   int word_length;
 
-  while (contraction_iterator_.GetNextWord(&word, &word_start, &word_length)) {
+  for (auto status =
+        text_iterator_.GetNextWord(&word, &word_start, &word_length);
```

Comment:
`text_iterator_` => `contraction_iterator_`

Reply:
Ooopy, sorry for my copy-pasta.